dtests/src/resources/scripts/cdcConnector/createAndLoadCDCMixedTables.sql

```diff
@@ -218,6 +218,7 @@ CREATE EXTERNAL TABLE IF NOT EXISTS  staging_UM_SERVICE
 CREATE TABLE IF NOT EXISTS UM_SERVICE USING column OPTIONS(partition_by 'UM_RVW_ID',redundancy '1',buckets '32',key_columns 'CLIENT_ID,UM_RVW_ID,UM_SERV_ID ' ) AS (SELECT * FROM staging_UM_SERVICE);
 
 DROP TABLE IF EXISTS INQUIRY;
+DROP TABLE IF EXISTS staging_INQUIRY;
 CREATE EXTERNAL TABLE IF NOT EXISTS  staging_INQUIRY
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data/TMG_RowTable_Data/INQUIRY.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
```

Comment:
Parametrize the variables in sql script and pass them through bt or conf file at runtime.

Reply:
Done